# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/python/hotfix-glibc-2.42.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

In file included from /usr/include/asm/ioctl.h:12,
                 from /usr/include/asm/ioctls.h:5,
                 from /usr/include/bits/ioctls.h:23,
                 from /usr/include/sys/ioctl.h:26,
                 from ./Modules/termios.c:13:
./Modules/termios.c:1119:16: error: invalid application of 'sizeof' to incomplete type 'struct termio'
 1119 |     {"TCGETA", TCGETA},
      |                ^~~~~~
./Modules/termios.c:1131:16: error: invalid application of 'sizeof' to incomplete type 'struct termio'
 1131 |     {"TCSETA", TCSETA},
      |                ^~~~~~
./Modules/termios.c:1134:17: error: invalid application of 'sizeof' to incomplete type 'struct termio'
 1134 |     {"TCSETAF", TCSETAF},
      |                 ^~~~~~~
./Modules/termios.c:1137:17: error: invalid application of 'sizeof' to incomplete type 'struct termio'
 1137 |     {"TCSETAW", TCSETAW},
      |                 ^~~~~~~

--- Python-3.13.6/Modules/termios.c.vanilla	2025-08-12 23:26:25.843474628 +0200
+++ Python-3.13.6/Modules/termios.c	2025-08-12 23:27:30.346400539 +0200
@@ -1115,7 +1115,7 @@
 #ifdef TCFLSH
     {"TCFLSH", TCFLSH},
 #endif
-#ifdef TCGETA
+#if 0
     {"TCGETA", TCGETA},
 #endif
 #ifdef TCGETS
@@ -1127,13 +1127,13 @@
 #ifdef TCSBRKP
     {"TCSBRKP", TCSBRKP},
 #endif
-#ifdef TCSETA
+#if 0
     {"TCSETA", TCSETA},
 #endif
-#ifdef TCSETAF
+#if 0 
     {"TCSETAF", TCSETAF},
 #endif
-#ifdef TCSETAW
+#if 0
     {"TCSETAW", TCSETAW},
 #endif
 #ifdef TCSETS
